js/contentAssessor.js

```diff
@@ -49,7 +50,7 @@ require( "util" ).inherits( ContentAssessor, Assessor );
  * @param {number} rating The rating to be weighted.
  * @returns {number} The weighted rating.
  */
-ContentAssessor.prototype.calculatePenaltyPointsEnglish = function( rating ) {
+ContentAssessor.prototype.calculatePenaltyPointsSupportedLanguage = function( rating ) {
```

Comment:
The function name is a little bit long and it is hard to understand what it does.